stable/metrics-server/templates/metrics-server-deployment.yaml

```diff
@@ -26,13 +26,65 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command:
             - /metrics-server
+            {{- if .Values.tlsConfig }}
+            - --requestheader-client-ca-file
+            {{- if .Values.tlsConfig.caFile }}
+            - --client-ca-file
+            - /certs/caFile
+            {{- end}}
+            {{- if  .Values.tlsConfig.certFile }}
+            - --tls-cert-file
+            - /certs/certFile
+            {{- end}}
+            {{- if .Values.tlsConfig.keyFile }}
+            - --tls-private-key-file
+            - /certs/keyFile
+            {{- end}}
+            {{- if .Values.tlsConfig.insecureSkipVerify }}
```

Comment:
This one shouldn't be a part of this config as it can be set independently of the other TLS settings

Reply:
insecureSkipVerify can be outside the  tls group but should i group them in a connection group?

I followed the prometheus-operator etcd example, maybe not the best one. I'm now making the changes to be similar to the prometheus-adapter as advised. will bring the insecureSkipVerify to outside and well see where we go from there

Reply:
it feels unnatural to be outside of the tls group as the flag it posts specifically says> - --kubelet-insecure-tls